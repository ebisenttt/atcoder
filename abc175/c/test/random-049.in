169607479675974 568817237814923 495405832616422
