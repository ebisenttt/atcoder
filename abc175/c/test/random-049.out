325798352940448
